Assets/AppCenter/AppCenterBehaviorAdvanced.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// Licensed under the MIT license.
+
+using System;
+using UnityEngine;
+
+[HelpURL("https://docs.microsoft.com/en-us/appcenter/sdk/crashes/unity")]
+public class AppCenterBehaviorAdvanced : MonoBehaviour
+{
+    public AppCenterSettingsAdvanced SettingsAdvanced;
+
+    private void Awake()
+    {
+        // Make sure that App Center have only one instance.
+        if (gameObject.GetComponent<AppCenterBehavior>() == null)
+        {
+            Debug.LogError("App Center should have the AppCenterBehavior instance attached to the game object!");
+            return;
+        }
+        DontDestroyOnLoad(gameObject);
+    }
+
+    private void Start()
```

Comment:
Do we really need this method?

Reply:
Why do we need it in `AppCenterBehavior` then?